content/en/news/2016-12-23-new-domain.md

```diff
@@ -0,0 +1,28 @@
+---
+title: New Domain Name
+date: 2016-12-23
+author: Sydney
+translator:
+summary: Info on our new Domain name
+---
+Hello all, hope everyone is having a happy holiday.
+
+On behalf of the Secretchronicles team, we are quite happy to announce that we have purchased
```

Comment:
You cannot combine "on behalf of" with "we" (except multiple persons wrote an article together and are speaking for an even greater multitude of persons). Either the team announces something (than "we"), or you announce something on behalf of the team (then "I").